test/tpmclient/tpmclient.int.c

```diff
@@ -602,7 +558,6 @@ static void TestHierarchyChangeAuth()
 #define PCR_16  16
 #define PCR_17  17
 #define PCR_18  18
-
 #define PCR_SIZE 20
```

Comment:
PCRs were 20 bytes in TPM1.2.  With TPM 2 it should be possible to have larger PCR values.  Is it worthwhile to test for longer PCRs.